timelock-corruption-detection/src/test/java/com/palantir/timelock/history/HistoryLoaderAndTransformerTest.java

```diff
@@ -73,11 +73,12 @@ public void setup() {
     public void canFetchLogsForQuery() {
         writeToLogs(1, 100);
         int lastVerified = 27;
-        List<HistoryQuery> historyQueries = ImmutableList.of(HistoryQuery.of(NAMESPACE_AND_USE_CASE, lastVerified));
+        List<HistoryQuery> historyQueries =
+                ImmutableList.of(HistoryQuery.of(NAMESPACE_AND_USE_CASE, lastVerified, lastVerified + 500));
         List<LogsForNamespaceAndUseCase> paxosHistory =
                 HistoryLoaderAndTransformer.getLogsForHistoryQueries(history, historyQueries);
 
-        sanityCheckLoadedHistory(paxosHistory, 100 - lastVerified);
+        sanityCheckLoadedHistory(paxosHistory, 100 - lastVerified + 1);
```

Comment:
Speaking of sanity checks, is this:

`(100 - lastVerified) + 1`
OR
`100 - (lastVerified + 1)`?

I assume the latter, although I'm never certain on how it determines this - so maybe bracket it?